test/e2e/cluster_upgrade_test.go

```diff
@@ -45,7 +45,7 @@ var _ = Describe("When upgrading a workload cluster using ClusterClass and testi
 		// or MachinePools are supported in ClusterClass.
 		version, err := semver.ParseTolerant(e2eConfig.GetVariable(KubernetesVersionUpgradeFrom))
 		Expect(err).ToNot(HaveOccurred(), "Invalid argument, KUBERNETES_VERSION_UPGRADE_FROM is not a valid version")
-		if version.LT(semver.MustParse("1.24.0")) {
+		if version.LT(semver.MustParse("1.25.0")) {
```

Comment:
I'm not sure this should be changed
@sbueringer opinions?

Reply:
Reading the code again I think you are right. It should be reverted.

Reply:
Exactly!